pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -172,9 +173,9 @@ public BaseSingleStageBrokerRequestHandler(PinotConfiguration config, String bro
     }
 
     LOGGER.info("Initialized {} with broker id: {}, timeout: {}ms, query response limit: {}, query log max length: {}, "
-            + "query log max rate: {}, query cancellation enabled: {}", getClass().getSimpleName(), _brokerId,
+            + "query log max rate: {}", getClass().getSimpleName(), _brokerId,
         _brokerTimeoutMs, _queryResponseLimit, _queryLogger.getMaxQueryLengthToLog(), _queryLogger.getLogRateLimit(),
-        enableQueryCancellation);
+        this.isQueryCancellationEnabled());
```

Comment:
Did you remove the `, query cancellation enabled: {}` part on propose? It looks like we have an extra parameter now.

Reply:
Can't say why and how, but it was unintentional. Already restored.